### PR TITLE
Add the community contribution and resources

### PR DESCRIPTION
**Pull request by shadowfall117**

### CODE REVIEW

File: machine_learning_tutorial.Rmd [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+library(randomForest)
+library(ggplot2)
+library(lattice)
+library(recipes)
+library(dplyr)
+library(AppliedPredictiveModeling)
+library(gridExtra)
+library(caTools)
+```
+## 1.Introduction
```
Comment (jontse) [inline machine_learning_tutorial.Rmd:23]: Please remove the numbered headings such as "1.", here and below.  The ## and ### will all convert into numbered headings during the rendering.

**Review by jontse — CHANGES_REQUESTED** [review]
Please remove the numbered headings such as "1." "2." below.  The ## and ### will all convert into numbered headings during the rendering.

**Review by shadowfall117 — COMMENTED** [review]
Hello, I have fixed the header issue

**Review by shadowfall117 — COMMENTED** [review]
Changed header

**Review by jontse — APPROVED** [review]
Thank you for updating and contributing!

**Comment by jontse** [comment]
Thank you for your contribution!

**Pull request merged**